examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
@maximede the hash syntax is fully optional. If you just use a single string it'll work like it always has :)

@jordansissel I'll add a :validate => lambda to it. Are there any good examples of it? What should it return? How should it behave?